app/views/nsm/steps/view_claim/allowed_disbursements.html.erb

```diff
@@ -5,45 +5,54 @@
     <%= t('.disbursements') %>
   </h3>
 
-  <% pagy, disbursements = pagy(@disbursements.where.not(adjustment_comment: nil)) %>
+  <%
+    pagy, disbursements = pagy_array(@disbursements.filter{_1.adjustment_comment.present?}, items: 100)
```

Comment:
It might be more performant to do this in the DB, i.e.:

```
pagy, disbursements = pagy(@disbursements.where.not(adjustment_comment:[nil, '']), items: 100)
```

Reply:
would have been nice, but `@disbursements` is a sorted array now rather than an activerecord relation